.openpublishing.publish.config.json

```diff
@@ -67,6 +67,18 @@
       "url": "https://github.com/MicrosoftDocs/azure-docs-pr",
       "branch": "main",
       "branch_mapping": {}
+    },
+    {
+      "path_to_root": "azure-dev-docs-pr",
```

Comment:
azure-management-docs-pr

Reply:
@tejaswikolli-web, azure-dev-docs-pr points to some of the files mentioned in the article and the dependent repos involved. Pls. confirm if you're asking to add a self-referring entry (azure-management-docs-pr) to the other blocks in the file.

Reply:
@LiSeda, @tejaswikolli-web is correct. You're in the public repo. You need to fork the private repo: https://github.com/MicrosoftDocs/azure-management-docs-pr

Reply:
@TomArcherMsft - As in move this whole thing to azure-management-docs-pr or add a block entry for this?

Reply:
@LiSeda You should always work in the private repo. This PR should be in azure-management-docs-pr.